src/styles/blocks/styles.scss

```diff
@@ -0,0 +1,44 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  width: $calendar-width;
+  gap: $gap-size;
+  align-self: center;
+  padding: $padding-size;
+  margin: 0;
+
+  &__day {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    width: $day-size;
+    height: $day-size;
+    background-color: $day-background;
+    border: $days-border-size;
+
+    &:hover {
+      background-color: $day-hover-background;
+      transform: translateY(-20px);
+      transition: 0.5s;
```

Comment:
use it on selector, not on `hover` only.
This advice was given to u by a another mentor in the past task, are u forget about it?